server/views/formPages/recat/securityBack.html

```diff
@@ -14,14 +14,18 @@
 
 <h1 id="securityInputHeader" class="govuk-heading-l">{{ pageTitle }}</h1>
 
-  {% set warningText %}
-    This prisoner was referred to the security team
-  {% endset %}
-  {{ govukWarningText({
-    text: warningText,
-    iconFallbackText: "Warning",
-    classes: "forms-warning-text warning-overrides"
-  }) }}
+  {% if (not data.socProfile.transferToSecurity) and (not data.extremismProfile.notifyRegionalCTLead) %}
```

Comment:
i think i note is for both initial and recat.

Reply:
thanks